### ISSUE TITLE
[meta] Rechartering 2025-2027

### ISSUE THREAD

**Issue by svgeesus** [w3c/csswg-drafts#10671]
The [current charter](https://www.w3.org/2023/01/css-2023.html) ends on 12 January 2025. So to get the ball rolling, here is a [proposed recharter](https://w3c.github.io/charter-drafts/2025/css-wg.html).

To avoid distracting diff-churn, the [list of deliverables](https://www.w3.org/2023/01/css-2023.html#deliverables) (with last publication dates, etc) is copy-pasted from the current charter. It will need to be regenerated just before this goes for AC review.

Here is a [diff to current charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2023%2F01%2Fcss-2023.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fcss-wg.html) and (mainly of use on the boilerplate-ey sections) [diff to template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fcss-wg.html)

The [scope](https://w3c.github.io/charter-drafts/2025/css-wg.html#scope) is unchanged, except that (to follow current practice) an introductory paragraph got moved into the Motivation and Background section.

The [decision policy](https://w3c.github.io/charter-drafts/2025/css-wg.html#decisions) is the same as the previous charter, too.

Basically this is a straightforward, "another two years" recharter.

I had a look at the [2023 timeline](https://www.w3.org/2023/01/css-2023.html#timeline) (the prediction of what modules would get to Recommendation in the charter period) and compared it to the actual situation. The [results](https://www.w3.org/Style/2024/CSS-2023-successes.html) could be better, some modules have had a bunch of work but have not been published to /TR _at all_ in the current charter period. Perhaps some of these could be republished before the charter period ends.

@astearns @atanassov

**Comment by Crissov** [comment]
I did not investigate why, but some links randomly display in a larger font size, at least on mobile Safari. This affects both the current and the proposed charter document.

**Comment by svgeesus** [comment]
Oh good catch, that was not an intentional removal.

**Comment by svgeesus** [comment]
> but the following text has been removed, and I'd rather it weren't:

Fixed.

**Comment by svgeesus** [comment]
> To avoid distracting diff-churn, the [list of deliverables](https://www.w3.org/2023/01/css-2023.html#deliverables) (with last publication dates, etc) is copy-pasted from the current charter. It will need to be regenerated just before this goes for AC review.

Charter now has [current deliverables list](https://w3c.github.io/charter-drafts/2025/css-wg.html#deliverables) including [Intersection Observer](https://w3c.github.io/charter-drafts/2025/css-wg.html#IntersectionObserver)

**Comment by svgeesus** [comment]
> I had a look at the [2023 timeline](https://www.w3.org/2023/01/css-2023.html#timeline) (the prediction of what modules would get to Recommendation in the charter period) and compared it to the actual situation. The [results](https://www.w3.org/Style/2024/CSS-2023-successes.html) could be better, some modules have had a bunch of work but have not been published to /TR _at all_ in the current charter period. Perhaps some of these could be republished before the charter period ends.

This has improved, but we still have a bunch of specs whose last publication was 2013, 2014 etc.

**Comment by svgeesus** [comment]
From the [AC Review form](https://www.w3.org/2002/09/wbs/33280/css-2025/) (Member-only link)

> Although most fonts used on web pages are delivered on demand (web fonts), fonts installed on the user's system remain important for legacy content, for support of minority languages, and to support rare or recently-encoded characters. However, exposing the set of installed fonts is a known and significant fingerprinting vector. There is thus a tension between the desire for privacy and the desire to render pages in some languages well (or indeed, at all).

> The CSS WG is working with the Privacy WG and the I18n WG to find an equitable solution to this problem.

**Comment by svgeesus** [comment]
Also from the review form (this was the discussion document for the TPAC 2024 breakout):

> A white paper, [Fonts, Privacy, and Not Breaking the Web](https://www.w3.org/2024/09/font-i18n-privacy.html)

**Comment by kbabbitt** [comment]
On the fonts/privacy/i18n issue, are there minutes or other notes captured from the discussion at TPAC that I could catch up on?

**Comment by svgeesus** [comment]
The meeting was held at lunch, after leaving the hotel due to a prolonged power cut. There are no minutes. 

That said: after a round of introductions (and summarizing the white paper, for those who had not had a chance to read it) the main new points coming out of the discussions were:

- not just minority languages are affected. Major languages, such as Chinese, will only render certain characters correctly if using the latest (user installed) fonts. And these are too large to use as webfonts
- preventing access to user-installed fonts is not just "this odd thing Safari started doing" but a good idea in general (reduced fingerprinting)
- providing opt-in access to user-installed fonts (either web-wide or, preferably, on a per-site basis) was necessary to correct the breakage that would otherwise occur (anything from a few wrong glyphs to the entire site being unreadable)
- as with all user permission prompts, thought is needed to cater for non-technical users

**Comment by svgeesus** [comment]
[New charter approved by AC 25 March 2025](https://www.w3.org/2025/03/css-wg.html)
